e2e/logInSignUp.spec.js

```diff
@@ -134,8 +135,22 @@ describe('Log in flow', () => {
 
 describe('testing swiping on feed page', () => {
   it('simulating swipes', async () => {
+    const searchSVG = await element(by.id('SearchSVG'));
+    const profileSVG = await element(by.id('ProfileSVG'));
+    const feedSVG = await element(by.id('FeedSVG'));
+    const uploadSVG = await element(by.id('UploadSVG'));
+
     // const swiperView = await getElementRef('feed-safe-area-view');
-    // elementIsVisible(swiperView);
+    await elementIsVisible(searchSVG);
+    await elementIsVisible(profileSVG);
+    await elementIsVisible(feedSVG);
+    await elementIsVisible(uploadSVG);
+
+    await(searchSVG).tap();
+    const logOut = await element(by.id('logout-button'));
```

Comment:
Visibility tests and the log out test could be separated into different sections (different description(...) or different it(...)), as they aren't directly related to each other